lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
```

Comment:
Basically, `localStorage` is part of the browser standard Javascript library so we can't move it to `lib/shared` because it's technically GUI. Unless we stop relying on `localStorage` through Redux in `lib/gui/models/store.js` we can't move any other models unless they're decoupled from `store.js`.